haiku-data/yab_documentation/yab_documentation-1.7.9.recipe

```diff
@@ -1,13 +1,13 @@
-SUMMARY="An integrated development environment for yab"
-DESCRIPTION="Documentaion for yab"
+SUMMARY="Documentation for the integrated development environment for yab"
+DESCRIPTION="Documentation for yab"
```

Comment:
If you change it to "Documentation for the yab IDE" it'll be found when searching for "IDE" in HaikuDepot.
So no new REVISION to force a rebuild of the package?

Reply:
This looks like it is documenting the programming language, not the IDE.

Reply:
Maybe:
```diff
-SUMMARY="Documentation for yab programming language"
-DESCRIPTION="Documentation for yab"
+SUMMARY="Documentation for yab"
+DESCRIPTION="Documentation for yab programming language and the yab IDE"
```